include/dlpack/dlpack.h

```diff
@@ -3,15 +3,18 @@
  * \file dlpack.h
  * \brief The common header of DLPack.
  */
-#ifndef DLPACK_H_
-#define DLPACK_H_
+#ifndef DLPACK_DLPACK_H_
+#define DLPACK_DLPACK_H_
 
 #ifdef __cplusplus
 #define DLPACK_EXTERN_C extern "C"
 #else
 #define DLPACK_EXTERN_C
 #endif
 
+/*! \brief The current version of dlpack */
+#define DLPACK_VERSION 010
```

Comment:
version number should go accord to
https://github.com/tqchen/dlpack/blob/master/CMakeLists.txt#L11